### DISCUSSION TITLE
Canada.ca design system

### DISCUSSION THREAD

**Discussion by shawnthompson** [gc-da11yn/gc-da11yn.github.io#5] [Inclusive Design (Open Discussions)]
The site will be using the [Canada.ca design system](https://www.canada.ca/en/government/about/design-system.html), which will give us the opportunity to report back any accessible issues we find.

**Comment by shawnthompson** [comment]
This site is a **campaign** for our work in accessibility within the Government of Canada.

**Reply by delisma** [reply]
For now how DTO would want to work is focussing on the consistency of the look and feel of Canada.ca and report any accessibility issues to Principal Publisher so that they can update GCWeb accordingly.

**Reply by shawnthompson** [reply]
I think us using Canada.ca look and feel will help us report back any accessibility issues we find because a site on Accessibility should be as accessible as possible.
